core/commonTest/src/contract/map/ImmutableMapTest.kt

```diff
@@ -283,13 +283,16 @@ abstract class ImmutableMapTest {
     }
 
     @Test fun noOperation() {
-        immutableMapOf<Int, String>().toPersistentMap().testNoOperation({ clear() }, { clear() })
+        immutableMapOf<Any, String>().toPersistentMap().testNoOperation({ clear() }, { clear() })
 
-        val map = immutableMapOf("x" to 1, null to "x").toPersistentMap()
+        val instance = Any()
+        val instance2 = Any()
+
+        val map = immutableMapOf("x" to instance, null to "x").toPersistentMap()
         with(map) {
             testNoOperation({ remove("y") }, { remove("y") })
-            testNoOperation({ remove("x", 2) }, { remove("x", 2) })
-            testNoOperation({ put("x", 1) }, { put("x", 1) })     // does not hold
+            testNoOperation({ remove("x", instance2) }, { remove("x", instance2) })
+            testNoOperation({ put("x", instance) }, { put("x", instance) })     // does not hold
             testNoOperation({ putAll(this) }, { putAll(this) })   // does not hold
```

Comment:
Could you please also remove the `// does not hold` comments? The checks should pass in all platforms now.